lib/install/validate/storage.go

```diff
@@ -41,11 +40,6 @@ func (v *Validator) storage(ctx context.Context, input *data.Data, conf *config.
 		return
 	}
 
-	// provide a default path if only a DS name is provided
-	if imageDSpath.Path == "" {
```

Comment:
So we used to do `path.Join( display name, "images" )`.  After this change, is the path just "display name"?  Because I would've expected line 46 to just be `imageDSpath.Path = input.DisplayName`